manifest.yaml

```diff
@@ -99,7 +99,7 @@ volumes:
     type: pointer
     package-id: "btc-rpc-proxy"
     volume-id: main
-    path: /root
+    path: /
```

Comment:
are you sure you want to do this?

Reply:
yes - the `path` in this case is relative to the where the `main` volume is already mounted at, which in this case is already `/root`.

Seeing if the path attribute can be omitted entirely if it's irrelevant (ie. `/`)

This is in response to this warning message in the EOS logs:

```
Jan 10 19:00:08 embassy-638b7e16 embassyd[2723483]: Jan 10 19:00:08.963  WARN execute{self=Docker(DockerAction { image: ImageId(Id("main")), system: false, entrypoint: "docker_entrypoint.sh", args: [], mounts: {Custom(Id("bitcoind")): "/mnt/bitcoind", Custom(Id("btc-rpc-proxy")): "/mnt/btc-rpc-proxy", Custom(Id("main")): "/root/.lnd"}, io_format: Some(Yaml), inject: false, shm_size_mb: None, sigterm_timeout: None }) pkg_id=PackageId(Id("lnd")) pkg_version=Version { version: Version(0, 13, 3, 1), string: "0.13.3.1" } name=None volumes=Volumes({Custom(Id("bitcoind")): Pointer { package_id: PackageId(Id("bitcoind")), volume_id: Custom(Id("main")), path: "/root", readonly: true }, Custom(Id("btc-rpc-proxy")): Pointer { package_id: PackageId(Id("btc-rpc-proxy")), volume_id: Custom(Id("main")), path: "/root", readonly: false }, Custom(Id("certificates")): Certificate { interface_id: InterfaceId(Id("control")) }, Custom(Id("compat")): Assets, Custom(Id("main")): Data { readonly: false }}) allow_inject=false timeout=None}:execute{self=DockerAction { image: ImageId(Id("main")), system: false, entrypoint: "docker_entrypoint.sh", args: [], mounts: {Custom(Id("bitcoind")): "/mnt/bitcoind", Custom(Id("btc-rpc-proxy")): "/mnt/btc-rpc-proxy", Custom(Id("main")): "/root/.lnd"}, io_format: Some(Yaml), inject: false, shm_size_mb: None, sigterm_timeout: None } pkg_id=PackageId(Id("lnd")) pkg_version=Version { version: Version(0, 13, 3, 1), string: "0.13.3.1" } name=None volumes=Volumes({Custom(Id("bitcoind")): Pointer { package_id: PackageId(Id("bitcoind")), volume_id: Custom(Id("main")), path: "/root", readonly: true }, Custom(Id("btc-rpc-proxy")): Pointer { package_id: PackageId(Id("btc-rpc-proxy")), volume_id: Custom(Id("main")), path: "/root", readonly: false }, Custom(Id("certificates")): Certificate { interface_id: InterfaceId(Id("control")) }, Custom(Id("compat")): Assets, Custom(Id("main")): Data { readonly: false }}) allow_inject=false timeout=None}: embassy::action::docker: /embassy-data/package-data/volumes/btc-rpc-proxy/data/main/root not mounted to container: No such file or directory (os error 2)

Reply:
the path attribute does need to exist in the manifest

Reply:
As @chrisguida suggested, it probably makes sense to just remove these pointer volumes. If this bug was present and nothing was broken, it seems like lnd was never using them.

Reply:
Looks like neither the `bitcoind` volume nor the `btc-rpc-proxy` volume are being used, let's remove them?

Reply:
Hah, beat me to it xD